frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/tags/OutTag.java

```diff
@@ -73,38 +72,35 @@ public int doStartTag() throws JspException {
                 }
             }
         }
-        return super.doStartTag();
-    }
 
-    //TODO remove unneeded...
-    private String escapeInDoubleQuote(String value) {
-        return value.replaceAll("\"", quoteReplacement("\\\""));
+        return super.doStartTag();
     }
 
-    //TODO remove unneeded...
-    protected String escapeInSingleQuote(String value) {
-        return value.replaceAll("'", quoteReplacement("\\\'"));
+    public void setValue(Object value) {
+        this.value = value;
     }
 
     public void setDefault(String def) {
```

Comment:
The `default` string is not used anymore? Can it be removed?

Reply:
The default string (def) is used in the super method call, so it should not be removed.